salt/utils/cloud.py

```diff
@@ -1205,7 +1216,10 @@ def deploy_windows(
         opts = {}
 
     if use_winrm and not HAS_WINRM:
-        log.error("WinRM requested but module winrm could not be imported")
+        log.error(
+            f"WinRM requested but module winrm could not be imported."
```

Comment:
Is this safe at the moment, with f-strings needing python 3.6 or higher?

Reply:
Probably not... Getting so used to using them I am forgetful to remove them. 😄